useful-scripts/get_data_block_responses/get_data_block_responses.py

```diff
@@ -22,18 +23,38 @@
 Place it in the same directory as this script.
 """
 
-DATA_API_URL = 'https://data.explore-education-statistics.service.gov.uk/api'
+parser = argparse.ArgumentParser(prog="python get_data_block_responses.py",
+                                 description="Used to get and time data block responses from "
+                                             "an environment")
+parser.add_argument("-u", "--url",
+                    dest="data_api_url",
+                    default="https://data.dev.explore-education-statistics.service.gov.uk/api",
+                    help="the Data API URL")
+parser.add_argument("-f", "--file",
+                    dest="datablocks_csv",
+                    default="dev_datablocks20201015.csv",
+                    help="CSV of data blocks (see comment in this script)")
+parser.add_argument("-r", "--resultsdir",
```

Comment:
With an `-e` flag, think we could remove this `-r` flag and just append the environment name to directory i.e. `results_dev` or `results_prod`, etc.